lightningd/channel_control.c

```diff
@@ -352,8 +360,8 @@ void peer_start_channeld(struct channel *channel,
 				      &channel->channel_info.remote_per_commit,
 				      &channel->channel_info.old_remote_per_commit,
 				      channel->funder,
-				      cfg->fee_base,
-				      cfg->fee_per_satoshi,
+				      fee_base,
```

Comment:
Were the previous commits failing the CI tests without this commit? If yes we need to re-arrange the commits so that no commit fails. This is important for bisectability, i.e., being able to test for a specific failure on a range of changes to find where a bug was introduced. If the commits in the commit history do not pass the regular tests, this is made a lot harder.

Reply:
thx, will do. I will completely redo the branch, force push here and we can hopefully do a cleaner second round :D